src/deriver/consumer.py

```
@@ -104,12 +107,27 @@ async def process_item(queue_item: models.QueueItem) -> None:
             await process_dream(validated, workspace_name)
+
+    elif task_type == "deletion":
+        with sentry_sdk.start_transaction(name="process_deletion_task", op="deriver"):
+            try:
+                validated = DeletionPayload(**queue_payload)
+            except ValidationError as e:
+                logger.error(
+                    "Invalid deletion payload received: %s. Payload: %s",
+                    str(e),
+                    queue_payload,
+                )
```

Ruff (0.14.7)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)